tools/cldr-apps/src/main/java/org/unicode/cldr/web/ReviewHide.java

```diff
@@ -103,8 +103,11 @@ private void getData() {
                 this.hiddenNotifications.put(subtype, xpstrid, val);
             }
         } catch (SQLException sqe) {
-            SurveyLog.logException(sqe, "Getting hidden notifications for uid#" + userId + " in " + localeId, null);
-            throw new InternalError("Error getting hidden notifications: " + sqe.getMessage());
+            // first version with cldr_dash_hide table is 42; silently ignore exceptions for earlier versions
+            if (Integer.parseInt(SurveyMain.getNewVersion()) >= 42) {
```

Comment:
I'd suggest something like `ReviewHide.FIRST_DB_VERSION`. 
In fact, even better might be:

`DBUtils.throwIfAtLeast(sqe, ReviewHide.FIRST_DB_VERSION, () -> "Getting hidden notifications for uid#" + userId + " in " + localeId)`

where that function checks `SurveyMain.getNewVersion()` (or via CLDRConfig) and throws an `InternalError` if it's a new version, otherwise does a `SurveyLog.warnOnce()`

the third arg to `throwIfAtLeast()` is a `Producer<String>` so only called when needed

I say this because I think we have and will have more of these.

But just externalizign the version number or doing nothing is fine too at your discretion

Reply:
I'll do a follow-up PR per your suggestions